src/Render.cc

```diff
@@ -57,13 +64,26 @@ namespace drafter {
             case JSONRenderFormat:
             {
                 refract::RenderJSONVisitor renderer;
-                refract::IElement* element = MSONToRefract(attributes);
+
+                refract::IElement* element = MSONToRefract(*attributes);
+
+                if (!element) {
+                    return body;
+                }
+
```

Comment:
OMG `ExpandRefract()` delete input element, but just in case while expansion happens?! It is really ugly behavior!!!